app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -387,6 +388,20 @@ class ProfileEditActivityTest {
     }
   }
 
+  @Test
+  fun testProfileEditActivity_clickProfileDeletionButton_configurationChange_dialogPersists() {
+    ActivityScenario.launch<ProfileEditActivity>(
+      ProfileEditActivity.createProfileEditActivity(
+        context,
+        1
+      )
+    ).use {
+      onView(withId(R.id.profile_delete_button)).perform(click())
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withText(R.string.profile_edit_delete_dialog_positive)).inRoot(isDialog())
```

Comment:
Are we not going to do any checking? 
Looks like incomplete test!!

Reply:
sorry forgot this. i thought inRoot gonna have a check inside

Reply:
It's a part of `ViewInteraction` - https://github.com/android/android-test/blob/master/espresso/core/java/androidx/test/espresso/ViewInteraction.java#L194

and as we know a test case has 3 steps:
1. ViewMatcher
2. ViewAction
3. ViewAssertion